pallets/author-mapping/src/lib.rs

```diff
@@ -364,10 +364,23 @@ pub mod pallet {
 		}
 	}
 
-	impl<T: Config> KeysLookup<NimbusId, T::Keys> for Pallet<T> {
+	impl<T: Config> KeysLookup<NimbusId, T::AccountId, T::Keys> for Pallet<T> {
 		fn lookup_keys(author: &NimbusId) -> Option<T::Keys> {
 			Self::keys_of(author)
 		}
+		/// Sets keys WITHOUT reserving deposit, for benchmarking purposes only
+		// #[cfg(feature = "runtime-benchmarking")] // will not compile if commented in
+		fn set_keys(id: NimbusId, account: T::AccountId, keys: T::Keys) {
+			use sp_runtime::traits::Zero;
+			MappingWithDeposit::<T>::insert(
+				id,
+				RegistrationInfo {
+					account,
+					deposit: Zero::zero(),
+					keys,
+				},
+			);
+		}
```

Comment:
I'll remove the comment before merging, but it did surprise me. So if the trait definition says this method is `#[cfg(feature = "runtime-benchmarking")]`, then the method definition does not require "`#[cfg(feature = "runtime-benchmarking")]`"?

Reply:
if you compile it without runtime-benchmarks, it wont ask for it. If you compile it with runtime-benchmarks, it will ask for it and you will need to implement it